Makefile

```diff
@@ -20,7 +20,7 @@ help: Makefile
 .PHONY: lint
 lint:
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45
-	golangci-lint run --build-tags integration --timeout 10m
+	golangci-lint run
```

Comment:
We don't want to have the integration build tag?

Reply:
It's in the .golangci.yml file :wink: 
No point having it twice :wink: